tests/utils/visit-block-page.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import {
+	createNewPost,
+	visitAdminPage,
+	insertBlock,
+	getEditedPostContent,
+} from '@wordpress/e2e-test-utils';
+import { outputFile } from 'fs-extra';
+import { dirname } from 'path';
+import kebabCase from 'lodash/kebabCase';
+
+async function visitPage( link ) {
+	await page.goto( link );
+	const isWelcomeGuideActive = await page.evaluate( () =>
+		wp.data.select( 'core/edit-post' ).isFeatureActive( 'welcomeGuide' )
+	);
+	const isFullscreenMode = await page.evaluate( () =>
+		wp.data.select( 'core/edit-post' ).isFeatureActive( 'fullscreenMode' )
+	);
+
+	if ( isWelcomeGuideActive ) {
+		await page.evaluate( () =>
+			wp.data.dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' )
+		);
+	}
+
+	if ( isFullscreenMode ) {
+		await page.evaluate( () =>
+			wp.data
+				.dispatch( 'core/edit-post' )
+				.toggleFeature( 'fullscreenMode' )
+		);
+	}
+}
+export async function visitBlockPage( title ) {
+	let link = '';
+	// Visit Import Products page.
+	await visitAdminPage( 'edit.php', 'post_type=page' );
+	// search for the page.
+	if ( await page.$( '#post-search-input' ) ) {
+		await page.type( '#post-search-input', title );
+		await page.click( '#search-submit', { waitUntil: 'domcontentloaded' } );
+		const pageLink = await page.$x( `//a[contains(text(), '${ title }')]` );
+		if ( ( await pageLink.length ) > 0 ) {
+			link = await page.evaluate(
+				( a ) => a.getAttribute( 'href' ),
+				pageLink[ 0 ]
+			);
```

Comment:
clicking the link directly seems to cause racing issues, so I opted for getting the link and passing it to `page.goto` as it handles waiting better.